e2e/scenario/scenario_ubuntu2204-custom-sysctls.go

```diff
@@ -35,7 +41,8 @@ func ubuntu2204CustomSysctls() *Scenario {
 				}
 			},
 			LiveVMValidators: []*LiveVMValidator{
-				SysctlConfigValidator(customSysctls),
```

Comment:
can we potentially just use a map[string]string and do type conversion as needed so we only need to call this a single time? each validator equates to another command that needs to be executed on the remote vm, so the fewer the better in terms of flakiness